device/marvell/armhf-marvell_et6448m_52x-r0/plugins/sfputil.py

```diff
@@ -113,8 +109,8 @@ def get_presence(self, port_num):
         pos = [1, 2, 4, 8]
         bit_pos = pos[prt]
         if smbus_present == 0:
-            cmdstatus, sfpstatus = commands.getstatusoutput(
-                'i2cget -y 0 0x41 0x3')  # need to verify the cpld register logic
+            p = subprocess.run(['i2cget', '-y', '0', '0x41', '0x3'], capture_output=True, universal_newlines=True)  # need to verify the cpld register logic
```

Comment:
>p [](http://example.com/codeflow?start=12&length=1)

LGTM actually catches a true bug in your code. You overwrite `p` and this is a behavior change.

Reply:
Suggest use a long name for one-shot usage in internal block. The less used, the longer the name.

Reply:
Right... I changed the variable name.